.github/workflows/docs.yml

```diff
@@ -12,8 +12,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Use rust nightly 2021-05-19 for building docs.
-      run: echo "nightly-2021-05-19" > ./rust-toolchain
+    - name: Use rust nightly 2021-08-24 for building docs.
+      run: echo "nightly-2021-08-24" > ./rust-toolchain
```

Comment:
We can drop these.  It was a temp hack to fix our published rust docs.  The sidebar wasn't showing our crates.